pkg/elasticetcd/config.go

```diff
@@ -69,18 +77,33 @@ type Config struct {
 	Endpoints, CURLs, PURLs types.URLs
 	IdealSize               int
 	DisableLogging          bool
+	UseTLS                  bool
+	CertFile, KeyFile       string
+	CAFile, TrustedCAFile   string
+	PeerCertFile            string
+	PeerKeyFile             string
+	ClntCertFile            string
+	ClntKeyFile             string
 }
 
 // NewConfig returns an ElasticEtcd config with defaults filled
 func NewConfig() *Config {
 	return &Config{
-		Name:      DefaultName,
-		Dir:       DefaultDir,
-		LogDir:    path.Join(DefaultDir, "log"),
-		Endpoints: defaultEndpoints,
-		CURLs:     defaultCURLs,
-		PURLs:     defaultPURLs,
-		IdealSize: DefaultIdealSize,
+		Name:          DefaultName,
+		Dir:           DefaultDir,
+		LogDir:        path.Join(DefaultDir, "log"),
+		Endpoints:     defaultEndpoints,
+		CURLs:         defaultCURLs,
+		PURLs:         defaultPURLs,
+		IdealSize:     DefaultIdealSize,
+		CertFile:      DefaultCertFile,
```

Comment:
It's not necessary for the cert files to be set in the default config, as they are not necessary for the operation of elasticetcd.

Reply:
Ack. Will remove these.